interface/forms/LBF/new.php

```diff
@@ -452,12 +715,45 @@ function validate(f) {
                         echo text($enrow['fname']) . ' ' . text($enrow['mname']) . ' ' . text($enrow['lname']);
                         echo ' ' . xlt('on') . ' ' . text(oeFormatShortDate(substr($enrow['date'], 0, 10))); ?>
                     </h3>
+<?php
+    $firow = sqlQuery("SELECT issue_id, provider_id FROM forms WHERE " .
+      "formdir = ? AND form_id = ? AND deleted = 0",
+      array($formname, $formid));
+    $form_issue_id = empty($firow['issue_id']) ? 0 : intval($firow['issue_id']);
+    $form_provider_id = empty($firow['provider_id']) ? 0 : intval($firow['provider_id']);
+
+    // Provider selector.
+    echo "&nbsp;&nbsp;";
+    echo xlt('Provider') . ": ";
+    // TBD: Refactor this function out of the FeeSheetHTML class as that is not the best place for it.
+    echo FeeSheetHtml::genProviderSelect('form_provider_id', '-- ' . xl("Please Select") . ' --', $form_provider_id);
+
+    // If appropriate build a drop-down selector of issues of this type for this patient.
+    // We skip this if in an issue form tab because removing and adding visit form tabs is
+    // beyond the current scope of that code.
+    if (!empty($LBF_ISSUE_TYPE) && !$from_issue_form) {
+      echo "&nbsp;&nbsp;";
+      $query = "SELECT id, title, date, begdate FROM lists WHERE pid = ? AND type = ? " .
+        "ORDER BY COALESCE(begdate, date) DESC, id DESC";
+      $ires = sqlStatement($query, array($pid, $LBF_ISSUE_TYPE));
+      echo "<select name='form_issue_id'>\n";
+      echo " <option value='0'>-- " . xlt('Select Case') . " --</option>\n";
+      while ($irow = sqlFetchArray($ires)) {
+        $issueid = $irow['id'];
+        $issuedate = oeFormatShortDate(empty($irow['begdate']) ? $irow['date'] : $irow['begdate']);
+        echo " <option value='$issueid'";
```

Comment:
attr() around $issueid

Reply:
It's always numeric but I'll change these to avoid anyone having to think about it.